gui/default/syncthing/settings/settingsModalView.html

```diff
@@ -327,7 +327,7 @@
     <button type="button" class="btn btn-primary btn-sm" ng-click="saveSettings()">
       <span class="fas fa-check"></span>&nbsp;<span translate>Save</span>
     </button>
-    <button type="button" class="btn btn-default btn-sm" data-dismiss="modal">
+    <button type="button" class="btn btn-default btn-sm" ng-click="closeSettings()">
       <span class="fas fa-times"></span>&nbsp;<span translate>Close</span>
```

Comment:
Clicking out of the modal also closes it, which would now no longer prompt users.